CHANGELOG

```diff
@@ -0,0 +1,109 @@
+# Changelog
+
+## [Unreleased]
+### New Features
+ * Support reapply configuration without link down when possible.
+ * Relicensed to LGPL 2.1+.
+ * Allow editing existing interface without type property in desire state.
+
+### Bug Fixes
+
+ * Allow adding slave to linux bridge with name only.
+ * Allow enabling IPv6 with empty address.
+ * netapplier: Initialize mainloop global object on failure.
```

Comment:
The front-end part is probably more interesting: Recover from an abnormal failure when libnmstate is re-used in a process (e.g. used from a daemonized service).